cinn/frontend/op_mappers/paddle/lookup_table.cc

```diff
@@ -29,7 +29,26 @@ void LookupTableOpMapper(const paddle::cpp::OpDesc& op_desc, const OpMapperConte
   auto w        = ctx.GetVar(w_name);
   auto ids      = ctx.GetVar(ids_name);
   CHECK(op_desc.HasAttr("padding_idx"));
-  auto padding_idx = op_desc.GetAttr<int64_t>("padding_idx");
+  auto padding_idx = static_cast<int64_t>(op_desc.GetAttr<int32_t>("padding_idx"));
```

Comment:
```suggestion
  auto padding_idx = utils::GetAttrOrDefault<int64_t>(op_desc, "padding_idx");
```
`GetAttrOrDefault`对int64_t做了额外处理，即使desc里存的是int32也可以获取。当然，上面的CHECK要保留。